### ISSUE TITLE
Separate scrapers into individual files

### ISSUE THREAD

**Issue by camjc** [lazd/coronadatascraper#97]
Its a bit of a pain for folks to contribute to the scraper since git will get super confused when trying to merge/rebase.

It'd be good to separate all the scraper definitions into their own files (maybe named by the country code or something) and then bring them all in at runtime, instead of just having one giant scraper file.

**Comment by lazd** [comment]
Totally agree. Now that priority and rating are implemented, order won't matter when it comes to overriding/duplicates, so we can dynamically import a whole directly of scrapers and run them.

**Comment by camjc** [comment]
```
import fastGlob from 'fast-glob';
export default async () => {
  const filePaths = await fastGlob(['./scrapers/**/*.js']);
  return Promise.all(
    filePaths.map(async filePath => import(filePath)))
      .then(modules => [
        ...modules.map(module => [module.default])
      ]
    );
};
```

**Comment by camjc** [comment]
Has anyone got a plan for splitting the file up?

was trying something like
```
const groupedScrapers = _(scrapers)
  .groupBy('country')
  .mapValues(countryArray =>
    _(countryArray)
      .groupBy('state')
      .mapValues(stateArray => _.groupBy(stateArray, 'county'))
      .value()
  )
  .value();

_.mapValues(groupedScrapers, (countryArray, country) => {
  console.log(countryArray, country);
  fs.writeFile(`./scrapers/${country}/index.js`, JSON.stringify(countryArray.undefined));
});
```


But not sure how to get the functions in-tact across the file writing. Open to any ideas (would hate to manually do this).

Above function goes toward the nested nature of /scrapers/COUNTRY/STATE/COUNTY where each level should end up being index.js for `undefined` of the groupBy.

**Comment by samirrayani** [comment]
@camjc I am looking into this now as well - my approach will be to use https://github.com/acornjs/acorn and https://github.com/estools/escodegen to parse the `scrapers.js` file and output a directory of scrapers as you suggested above

```
/scrapers
└── US
    ├── FL
    │   ├── BROWARD
    │   │   └── index.js
    │   ├── DADE
    │   │   └── index.js
    │   └── index.js
    └── index.js
```

I'm working on a solution now and will update you as I make more progress

**Comment by camjc** [comment]
Ah, of course. Smart! Obviously feel free to use any of my snippets above if at all helpful.  Good luck!

**Comment by lazd** [comment]
@samirrayani awesome.

Thoughts on putting them in subdirectories vs a single directory with `CA-USA.js` and `Los Angeles County-CA-USA.js`? Better or worse?

**Comment by samirrayani** [comment]
we have flexibility wrt directory structure; with >3k counties, I'm tempted to do subdirectories so it's easy to navigate but that _might_ be overkill. some sort of top-level README could also help facilitate this

**Comment by qgolsteyn** [comment]
Closed with #137
